### DISCUSSION TITLE
Problems listing data

### DISCUSSION THREAD

**Discussion by gerickt** [Budibase/budibase#9490] [Help]
My data provider calls a MySQL table.
In limit, I put 515 (the number of existing records)
But I only get 266 items in the Options Picker.

Regardless of its increase or decrease of the limit of the sort order, it only shows half of the results, I have the same problem with other more extensive tables.

What could be the issue?

**Comment by melohagan** [comment]
It is possibly related to this: https://github.com/Budibase/budibase/issues/8558

You could try increasing the SQL_MAX_ROWS and see if that helps. This would assume that table has a relationship defined?
